tests/implied-strict.js

```diff
@@ -108,8 +108,7 @@ describe("impliedStrict option", () => {
 
     it("omits a module global scope when ensuring all user scopes are strict", () => {
         const ast = espree(`
-            function foo() {}`
-        );
+            function foo() {}`);
```

Comment:
Maybe this should just be one line?

```js
const ast = espree(`function foo() {}`);
```

Reply:
This was from the autofixer - good catch!